lib/pf/UnifiedApi/Controller/Config/Roles.pm

```diff
@@ -0,0 +1,59 @@
+package pf::UnifiedApi::Controller::Config::Roles;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Config::Roles - 
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Config::Roles
+
+
+
+=cut
+
+use strict;
+use warnings;
+
+
+use Mojo::Base qw(pf::UnifiedApi::Controller::Config);
+
+has 'config_store_class' => 'pf::ConfigStore::Roles';
+has 'form_class' => 'pfappserver::Form::Config::Roles';
+has 'primary_key' => 'role_id';
+
+use pf::ConfigStore::Roles;
+use pfappserver::Form::Config::Roles;
+
```

Comment:
deletion should be prevented here as well as renaming if possible but AFAIK renaming is not possible through a single API call so just deletion prevention should be golden